### PR TITLE
Example files sprint 1.2

### PR DESCRIPTION
**Pull request by larswik**
## File

Example files for sprint 1.2 from Lars W

## Covered Concept Templates - Usages Included

1.  Project Global Positioning (A)
2. Project Global Positioning Geodetic (B)
3. Project Global Positioning Mapped (C)
4. Classification Information
5. Spatial Decomposition
6. Project declaration (IfcBuiltSystem/TUNNEL_SUPPORT)

### CODE REVIEW

File: files/project_team/IFCTunnelDep1.2_B.ifc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#18= IFCRELAGGREGATES('3IsnM3y1aDC$votwM3D_wZ',#10,$,$,#11,(#4));
+#19= IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#20= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#21= IFCDIRECTION((0.0,0.0,1.0));
+#22= IFCDIRECTION((1.0,0.0,0.0));
+#23= IFCAXIS2PLACEMENT3D(#20,#21,#22);
+#24= IFCDIRECTION((1.0,0.0,0.0));
+#25= IFCGEOMETRICREPRESENTATIONCONTEXT('3D','Model',3,0.0000000254,#23,#26);
+#26= IFCDIRECTION((0.0,1.0));
+#27= IFCGEOGRAPHICCRS('EPSG:4326',$,,$,$,$);
```
Comment (jmirtsch) [inline files/project_team/IFCTunnelDep1.2_B.ifc:44]:
```suggestion
#27= IFCGEOGRAPHICCRS('EPSG:4326',$,$,$,$);
```
Manually Fix Serialization bug.

**Comment by pjanck** [comment]
At the request of PM, I've included Sprint 1.1 checks in the checking procedure as well. Thus, all examples fail because of missing Project Declaration. Otherwise the checks should be green - @larswik care to double-check?

**Comment by larswik** [comment]
> At the request of PM, I've included Sprint 1.1 checks in the checking procedure as well. Thus, all examples fail because of missing Project Declaration. Otherwise the checks should be green - @larswik care to double-check?

Added an IfcBuiltSystem/TUNNEL_SUPPORT declaration and at least the files passed all tests and the tests also discovered the correct georeferencing cases. Please review and merge if ok.

**Review by pjanck — APPROVED** [review]
lftm

**Pull request merged**